src/main/daml/Daml/Finance/Interface/Types/Common.daml

```diff
@@ -59,4 +59,4 @@ data Quantity u a = Quantity
       -- ^ The quantity's unit.
     amount : a
       -- ^ A numerical amount.
-  deriving (Eq, Show)
+  deriving (Eq, Ord, Show)
```

Comment:
Is this required? I wonder what the default derivation is for this type

Reply:
It was required because the `View` data type for the `Effect` also had `Ord` derived. It's not clear why we would need to have order derived but adding `Ord` keeps the view consistent with all the other defined `View` types.

It's probably something we'd want to review...